### PR TITLE
ldap: add tests for udp and frames

### PR DESCRIPTION
**Pull request by glongo**
## Ticket

If your pull request is related to a Suricata ticket, please provide
the full URL to the ticket here so this pull request can monitor
changes to the ticket status:

Redmine ticket:
https://redmine.openinfosecfoundation.org/issues/1199

### CODE REVIEW

File: tests/ldap-frames/test.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+pcap: ../ldap-udp/cldap.pcap
+
+checks:
+  - filter:
+      count: 1
+      match:
+        pcap_cnt: 1
+        event_type: frame
+        frame.direction: toserver
```
Comment (catenacyber) [inline tests/ldap-frames/test.yaml:15]: You need a suricata.yaml to enable frames output ;-)

File: tests/ldap-udp/test.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+requires:
+  min-version: 8
+
+args:
+  - -k none
+
+pcap: cldap.pcap
```
Comment (catenacyber) [inline tests/ldap-udp/test.yaml:7]: no need to specify the pcap in the same directory ;-)

**Review by victorjulien — APPROVED** [review]

File: tests/ldap-frames/test.yaml (hunk 1 above)
Comment (catenacyber) [reply tests/ldap-frames/test.yaml:15]: And you added one :-)

**Review by victorjulien — APPROVED** [review]

**Comment by victorjulien** [comment]
Merged in #2003, thanks!

**Pull request closed without merging**